gazebo_plugins/scripts/wheel_slip_drawbar_pull_publisher.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Desc: helper script for publishing drawbar pull wrenches
+# Author: Steve Peters
+#
+import argparse
+import math
+import rclpy
+import sys
+from geometry_msgs.msg import Wrench
+from rclpy.node import Node
+
+DEFAULT_FORCE_INCREMENT = 10
+DEFAULT_MAX_FORCE = 70
+DEFAULT_PUBLISH_INTERVAL = 2.0
+
+class WheelSlipDrawbarPullPublisher(Node):
+    def __init__(self, args):
+        super().__init__('wheel_slip_drawbar_pull_publisher')
+        parser = argparse.ArgumentParser(
+            description='Publish drag forces as a wrench representing drawbar pull on the back of a vehicle.')
+        parser.add_argument('-f', '--force-increment', type=float, default=DEFAULT_FORCE_INCREMENT,
+                            help='The drawbar pull force increment (Newtons)')
+        parser.add_argument('-i', '--interval', type=float, default=DEFAULT_PUBLISH_INTERVAL,
```

Comment:
Is there a way to make this timer trigger on the basis of simtime ? Currently, these intervals are not "true time intervals" for the simulation, and results might vary based on system load.

Reply:
I guess this is still fine since ``/drawbar_pull`` is being published explicitly :+1: